reduction/imaging_parameters.py

```diff
@@ -2973,22 +2910,11 @@
         "threshold": "16mJy",  # sigma is ~4 mJy
         "startmodel": "W51-E_B6_uid___A001_X1296_X213_continuum_merged_12M_robust0_selfcal7_finaliter",
     },
-    "W51-E_B6_12M_robust0_contsub": {
-        "usemask": "auto-multithresh",
-        "sidelobethreshold": 1.0,
-        "pblimit": 0.1,
-        "threshold": "16mJy",  # sigma is ~4 mJy
-    },
     "W51-E_B3_12M_robust0": {
         "startmodel": "W51-E_B3_uid___A001_X1296_X10b_continuum_merged_12M_robust0_selfcal4",
         "threshold": "4mJy",  # sigma is ~0.8 mJy
         "pblimit": 0.05,  # per Nov 6 telecon
     },
-    "W51-E_B3_12M_robust0_contsub": {
-        "threshold": "4mJy",  # sigma is ~0.8 mJy
-        "sidelobethreshold": 1.0,
-        "pblimit": 0.05,
-    },
```

Comment:
these look like they should not have been changed

Reply:
They should not be changed if somebody else (you?) is making contsub cubes, but I thought only I was interested (for now) in such cubes. The one with "usemask": "auto-multithresh" for "W51-E_B6_12M_robust0_contsub" should be most certainly me and should be deleted. The one for "W51-E_B3_12M_robust0_contsub" probably should be left, in case you were trying custom contsub images.

Reply:
no, you're right, I'll clear the contsub stuff